tests/TMDBApp.UnitTests/Services/MovieServicesTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using TMDBApp.Core.Services.Genres;
+using TMDBApp.Core.Services.Movies;
+using TMDBApp.Core.Services.RequestsProvider;
+using TMDBApp.Core.Services.Settings;
+using Xunit;
+
+namespace TMDBApp.UnitTests.Services
+{
+    public class MovieServicesTest
+    {
+        [Fact]
+        public async Task GetUpcomingMovieShouldReturnItens()
```

Comment:
Improve test description because it is also testing the release date. It also should verify if the ReleaseDate is >= than DateTime.Now.Date.